src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,88 @@
+REM
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+REM %3: Response files location
+
+@echo off
+setlocal
```

Comment:
NIT `@echo off` should be at the beginning of the script, and you should be showing usage info only when you need to (for example, not enough params passed) to prevent cluttering up any console output